lib/tasks/daily_summary.rake

```diff
@@ -0,0 +1,28 @@
+namespace :daily_summary do
+  desc "Send daily summary emails to users directly related to issue changes"
+  task send_emails: :environment do
+    organizations = Organization.all
+    organizations.find_each do |organization|
+      issues = IssueHistory.where('created_at >= ? AND organization_id = ?', 1.day.ago, organization.id)
```

Comment:
Is there any other way of doing this rather than putting the where of organization id